help/C/index.docbook

```diff
@@ -1427,41 +1442,107 @@
+      <title>File Systems</title>
+        <variablelist>
+          <varlistentry>
+            <term><guilabel>File Systems</guilabel></term>
+            <listitem>
+              <itemizedlist>
                 <listitem>
                   <para>
-                    <guilabel>Grid color</guilabel>
+                    <guilabel>Update interval in seconds</guilabel>
                   </para>
                   <para>
-                    Use this button to customize the grid color of the <application>&app;</application> graphs, as described in <xref linkend="mate-system-monitor-customize-grid"/>.
+                    Use this spin box to specify how often you want to update the <guilabel>File Systems</guilabel> table.
+                  </para>
+                </listitem>
+                <listitem>
+                  <para>
+                    <guilabel>Show all file systems</guilabel>
+                  </para>
+                  <para>
+                    Select this option to show all file systems, including temporary and system ones.
                   </para>
                 </listitem>
               </itemizedlist>
             </listitem>
           </varlistentry>
           <varlistentry>
-            <term><guilabel>File Systems</guilabel></term>
+            <term>
+              <guilabel>Information Fields</guilabel>
+            </term>
             <listitem>
+              <para>
+                Use the following options to select which fields are displayed in the file system list:
+              </para>
               <itemizedlist>
                 <listitem>
                   <para>
-                    <guilabel>Update interval in seconds</guilabel>
+                    <guilabel>Device</guilabel>
                   </para>
                   <para>
-                    Use this spin box to specify how often you want to update the <guilabel>File Systems</guilabel> table.
+                    Select this option to display the location of the block file for the device.
                   </para>
                 </listitem>
                 <listitem>
                   <para>
-                    <guilabel>Show all file systems</guilabel>
+                    <guilabel>Directory</guilabel>
                   </para>
                   <para>
-                    Select this option to show all file systems, including temporary and system ones.
+                    Select this option to display the mount point (directory to access) of a device.
+                  </para>
+                </listitem>
+                <listitem>
+                  <para>
+                    <guilabel>Type</guilabel>
+                  </para>
+                  <para>
+                    Select this option to display the file system type. Linux supports numerous file system types like NTFS, FAT32, ext4, ext3 ext2 etc. If the file system is of any supported type, it will be shown if this option is selected.
```

Comment:
It's only an example, you can rewrite the paragraph. (This application runs in other operating systems and Linux is just a kernel name)

> In most GNU/Linux distributions and other Unix-like operating systems, there are numerous file system types which are supported by the kernel, like NTFS, FAT32, ext4, ext3, ext2, etc. The file system type will be shown if this option is selected in the case of a supported type.